test/MOI_wrapper.jl

```diff
@@ -78,3 +78,30 @@ end
         "solve_objbound_edge_cases"
     ])
 end
+
+@testset "Test params" begin
+    default_model = ModelForCachingOptimizer{Float64}()
+    x = MOI.add_variables(default_model, 100)
+    terms = [MOI.ScalarAffineTerm(rand(), x[i]) for i in 1:length(x)]
```

Comment:
I wouldn't recommend using random coefficients because that makes it hard to reproduce test failures.